src/kernel.cpp

```diff
@@ -37,6 +37,7 @@ static std::map<int, unsigned int> mapStakeModifierCheckpoints =
     ( 30583, 0xdc7bf136u )
     ( 99999, 0xf555cfd2u )
     (219999, 0x91b7444du )
+    (336000, 0x6c3c8048 )
```

Comment:
All the other hex values are unsigned, shouldn't this one be unsigned too?

Reply:
Good catch :+1:  It is fixed now :)